apps/site/src/components/Footer/Footer.module.scss

```diff
@@ -11,5 +11,9 @@
 }
 
 .socials {
-	margin: 10px;
+	background-color: white;
+	margin: 15px;
+	border: 5px solid white;
+	border-radius: 10px;
+	filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.25));
```

Comment:
Issue: It's a bit hard to tell where the "sticker" ends on the top. It might be good to have a shadow evenly surrounding all sides instead, so that we can better see the sticker and also somewhat address @tyleryy 's suggestion for a dark border.

Reply:
This will create a dark border, but if we plan to put animations on it (say onHover the image scales to give the illusion of lifting up like a sticker), the dark border from the drop shadow will disappear. Just something to consider.

Reply:
That's a good point. However, since the deadline is coming up and classes are beginning tomorrow, we can open a separate issue to address this.